core/templates/dev/head/dashboard/Dashboard.js

```diff
@@ -71,14 +73,8 @@ oppia.controller('Dashboard', [
     $rootScope.loadingMessage = 'Loading';
     DashboardBackendApiService.fetchDashboardData().then(
       function(response) {
-        $scope.explorationsList = response.explorations_list.sort(
-          function(a, b) {
-            return (a.title === '' ? 1 :
-              b.title === '' ? -1 :
-              a.title < b.title ? -1 :
-              a.title > b.title ? 1 : 0);
-          }
-        );
+        $scope.explorationsList = (
+          sortExplorationsService.sortBy(response.explorations_list, 'title'));
```

Comment:
Hm, is this related to the email I wrote some time back (titled "Requests for the creator dashboard") that said:

- More generally, I feel like one thing that's missing for me is a way to automatically surface which of my explorations "need attention" and what I should do with them to get them back into a "healthy" state. It would be nice if you could make this easy for me!